rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1030,7 +1030,7 @@ def try_insert_mapping(cur):
         oracles_after_upgrade = cursor.execute(
             'SELECT value FROM settings WHERE name="historical_price_oracles"',
         ).fetchone()[0]
-        assert oracles_after_upgrade == '["manual", "cryptocompare", "coingecko", "defillama"]'
+        assert oracles_after_upgrade == json.dumps([oracle.serialize() for oracle in DEFAULT_HISTORICAL_PRICE_ORACLES_ORDER])  # noqa: E501
```

Comment:
`DEFAULT_HISTORICAL_PRICE_ORACLES_ORDER` is used to set the value in the [old upgrade](https://github.com/rotki/rotki/blob/develop/rotkehlchen/db/upgrades/v34_v35.py#L362). 
Now we've changed `DEFAULT_HISTORICAL_PRICE_ORACLES_ORDER` to include the uniswap oracles and so this test fails, since it was hardcoded in the test and not the upgrade.

Reply:
can you modify the old upgrade to have the values hardocoded? It can have unexpected behavior  in old upgrades to have new values